proto/plan_common.proto

```diff
@@ -16,14 +16,15 @@
~  string name = 2;
~}
~
~enum ColumnDescVersion {
~  COLUMN_DESC_VERSION_UNSPECIFIED = 0;
~  // Introduced in https://github.com/risingwavelabs/risingwave/pull/13707#discussion_r1429947537,
   // in case DEFAULT_KEY_COLUMN_NAME changes
   COLUMN_DESC_VERSION_PR_13707 = 1;
 
-  // for test only
-  COLUMN_DESC_VERSION_MAX = 2147483647;
```

Comment:
Removing `MAX` doesn't matter because this should only be used in unit tests before.